lib/rex/post/meterpreter/ui/console/command_dispatcher.rb

```diff
@@ -50,6 +50,23 @@ def client
   end
 
   #
+  # Returns the commands that meet the requirements
+  #
+  def check_commands(all, reqs=nil)
+    all.delete_if do |cmd, _desc|
+      del = false
+      reqs[cmd].each do |req|
```

Comment:
Could this be a problem if `reqs` is actually nil? Perhaps line `55` should default to an empty hash instead of `nil` ? Perhaps not having a default param here would be better?